admin-manual/installation/linux/ubuntu-bionic.rst

```diff
@@ -60,11 +66,16 @@ Ubuntu doesn't provide a package but you can download it directly from the
 `Elasticsearch site <https://www.elastic.co/downloads/elasticsearch>`_ if you
 are unable to download it using the method that follows.
 
-First, make sure that `Java <https://www.java.com/en/>`__ is installed. In this
+Make sure that `Java <https://www.java.com/en/>`__ is installed. In this
 example we are going to use OpenJDK but Oracle's JVM would also work.
 
+   code content
+
 .. code-block:: bash
 
+   sudo add-apt-repository ppa:openjdk-r/ppa
+   sudo apt-get update
+
```

Comment:
It's hard to tell in the diff, but it looks like there might be an extra blank linebreak between the apt-get update and the apt install openjdk command?